XenAdmin/Wizards/PatchingWizard/PatchingWizard_PatchingPage.cs

```diff
@@ -207,29 +205,6 @@ private List<HostPlan> CompileManualHostPlan(Pool pool, out List<Host> applicabl
                             hostplans.Add(new HostPlan(server, null, updateActions, null));
                         }
                 }
-                else if (SuppPackVdis != null) //supp pack
-                {
-                    foreach (var server in SelectedServers)
-                        if (SuppPackVdis.ContainsKey(server) && poolHosts.Contains(server))
-                        {
-                            var updateActions = new List<PlanAction> {new InstallSupplementalPackPlanAction(server, SuppPackVdis[server])};
-                            hostplans.Add(new HostPlan(server, null, updateActions, null));
-                        }
-                }

```

Comment:
we should at a minimum log that `PoolUpdate` was null here. Maybe even a `Debug.Assert`? Since it's not supposed to happen